add_student.html

```diff
@@ -1,13 +1,62 @@
-<!--
-This file will provide the backbone for adding info for a single student.
--->
-<html>
-  <head>
-  <title>Add Student</title>
-  <link rel="stylesheet" type="text/css" href="css/upls_styles.css">
-  <script src="js/main_menu.js"></script>
-  </head>
-  <body onload="mainMenu()">
-    <div class="main_menu" id="main_menu"></div>
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Add Student</title>
+    <script type="text/javascript"></script>
+    <script src="https://apis.google.com/js/api.js"></script>
+    <script src="Api.js"></script>
+</head>
+<body>
+<script language="javascript">
+    let sheetID = "";
+    let sa = new SheetsApi("","","");
+    sa.handleClientLoad();

```

Comment:
This should be updated to comply with the SheetsApi instantiation without parameters. See [Zeru's code](https://github.com/upeipls/UPLS/blob/45d665b05e462d74dbed6a3b07ec79b35c361c02/js/edit_header.js#L1-L10) and implement something like that in your js files.